src/qt/rpcconsole.cpp

```diff
@@ -514,8 +515,17 @@ RPCConsole::RPCConsole(interfaces::Node& node, const PlatformStyle *_platformSty
         /*: Explanatory text for a short-lived outbound peer connection that is used
             to request addresses from a peer. */
         tr("Outbound Address Fetch: short-lived, for soliciting addresses")};
-    const QString list{"<ul><li>" + Join(CONNECTION_TYPE_DOC, QString("</li><li>")) + "</li></ul>"};
-    ui->peerConnectionTypeLabel->setToolTip(ui->peerConnectionTypeLabel->toolTip().arg(list));
+    const QString connection_types_list{"<ul><li>" + Join(CONNECTION_TYPE_DOC, QString("</li><li>")) + "</li></ul>"};
+    ui->peerConnectionTypeLabel->setToolTip(ui->peerConnectionTypeLabel->toolTip().arg(connection_types_list));
+    const std::vector<QString> TRANSPORT_TYPE_DOC{
+        //: Explanatory text for "detecting" transport type.
+        tr("detecting: incoming connection; peer could be v1 or v2"),
```

Comment:
In the latest version of https://github.com/bitcoin/bitcoin/pull/28331 (since the change in https://github.com/bitcoin/bitcoin/pull/28331#issuecomment-1721167073 was applied), the "detecting" state can appear in both directions, i.e. the description should be adapted accordingly:
```suggestion
        tr("detecting: peer could be v1 or v2"),
```

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/754#issuecomment-1732614324).